EPRN.Portal/Services/WasteService.cs

```diff
@@ -79,6 +79,56 @@ public async Task<WasteTypesViewModel> GetWasteTypesViewModel(int journeyId)
             };
         }
 
+        public async Task<WasteSubTypesViewModel> GetWasteSubTypesViewModel(int journeyId)
+        {
+            var wasteTypeId = await _httpJourneyService.GetWasteTypeId(journeyId);
+
+            var wasteMaterialSubTypes = await _httpWasteService.GetWasteMaterialSubTypes(wasteTypeId);
```

Comment:
Don't await twice. Asign both these as tasks and do a `await Task.WhenAll(task1, task2)`